src/entities/Snapshot/utils.ts

```diff
@@ -87,3 +91,65 @@ export function groupProposalsByMonth(proposals: Partial<SnapshotProposal>[], fi
 
   return data
 }
+
+export async function getSnapshotStatusAndSpace(spaceName?: string) {
```

Comment:
Shouldn't this be part of `SnapshotService` or is it because it is used by the frontend?

Reply:
Yeah, it's used in the frontend. When I moved it to the Service it wouldn't work. 
I'm not sure how to make a file available to the front from a different path.